pkgs/development/python-modules/pex/default.nix

```diff
@@ -2,23 +2,26 @@
   lib,
   buildPythonPackage,
   fetchPypi,
-  hatchling,
+  setuptools,
   pythonOlder,
 }:
-
 buildPythonPackage rec {
```

Comment:
Please reorder the arguments to `buildPythonPackage` as follows:

`pname`, `version`, `pyproject`, `src`, `patches`, `build-system`, `pythonImportsCheck`, `doCheck`, `meta`.
`disabled` can be removed, as it currently does nothing.

Reply:
This PR does neither touch the order nor the attribute, so it doesn't change the status quo here.
You are free to open a separate PR or invest time into writing either policy or ideally an automated check if this is something you care about.

Reply:
Ah I see, argument order should only be done in new packages or reordered in refactors.

Reply:
It should either be done on a best effort basis or be enforced by an automated check + tree-wide change at introduction.
Commenting on minor aesthetics during manual reviews is a waste of time IMO as it scales quite badly.